### ISSUE TITLE
`test_model_definitions_match_ddl` fails because of an extra mock resource table

### ISSUE THREAD

**Issue by fivegrant** [schireson/pytest-alembic#75]
I'm currently adding migrations to my poetry project using alembic + pytest-alembic by creating a single alembic version and everything is working well except for one test:
> pytest_alembic.plugin.error.AlembicTestFailure: The models describing the DDL of your database are out of sync with the set of steps described in the revision history. This usually means that someone has made manual changes to the database's DDL, or some model has been changed without also generating a migration to describe that change.

Specifically, the issue seems to be that the postgres mock resource creates an additional table that obviously not part of my sqlalchemy orms. 
```
# ### commands auto generated by Alembic - please adjust! ###
op.drop_table('pytest_mock_resource_db')
# ### end Alembic commands ###
```

Is there a way to make this test ignore the table or ensure `PostgresConfig` doesn't create this table? 

```python3
# tests/migrations.py
from pytest import fixture
from _pytest.assertion import truncate
from pytest_alembic.tests import (
    test_model_definitions_match_ddl, # this test fails
    test_single_head_revision, # works!
    test_up_down_consistency, # works!
    test_upgrade, # works!
)
from pytest_mock_resources import PostgresConfig, create_postgres_fixture


@fixture(scope="session")
def pmr_postgres_config():
    """
    Match postgres to image used in the docker compose and the
    expectations in `alembic.ini`
    """
    return PostgresConfig(
        image="postgres:15.1",
        port=5433,
        ci_port=5433,
        username="dev",
        password="dev",
        root_database="proj",
    )


@fixture
def alembic_config():
    return {"file": "migrate/alembic.ini"} # i'm storing alembic in the same subdirectory with `versions` and `env.py`


alembic_engine = create_postgres_fixture()
```

**Comment by DanCardin** [comment]
tl;dr you dont want to be connecting to the `root_database`, and there's likely no value to you in configuring it in the above fixture.

---
You appear to be connecting to the `root_database` in alembic. The PMR-root database includes plumbing that PMR depends upon to function.

Some context: internally PMR is creating a new database per test and handing it to consumers. That's how it avoids inter-test state issues. So it's important to avoid connecting to a specific database name, unless the tests are inherently guaranteed to not leave any state behind.

This ["connectable" section](https://pytest-alembic.readthedocs.io/en/latest/setup.html#env-py) is particularly important for `env.py` to function well with PMR. Namely this part:
```
connectable = context.config.attributes.get("connection", None)
if connectable is None:
    ... # whatever you do normally
```
where the config attributes' "connection" is runtime configured by pytest-alembic for you to use in the env.py.

**Comment by fivegrant** [comment]
Okay, that makes sense. Thanks for the help!
